### PR TITLE
Editing pass for spring-batch-integration.adoc

### PR DESCRIPTION
**Pull request by Buzzardo**
I improved readability and consistency and fixed sentence errors. No questions in this one.

### CODE REVIEW

File: spring-batch-docs/asciidoc/spring-batch-integration.adoc [hunk 1]
```diff
@@ -161,16 +161,16 @@ Spring Batch Integration provides the
 `JobLaunchingMessageHandler` class that you can
 use to launch batch jobs. The input for the
 `JobLaunchingMessageHandler` is provided by a
-Spring Integration message, which payload is of type
-JobLaunchRequest. This class is a wrapper around the Job
-    that needs to be launched as well as the JobParameters
+Spring Integration message, which has a payload of type
+`JobLaunchRequest`. This class is a wrapper around the Job
```
Comment (mminella) [inline spring-batch-docs/asciidoc/spring-batch-integration.adoc]: Tics around Job

File: spring-batch-docs/asciidoc/spring-batch-integration.adoc [hunk 2]
```diff
@@ -315,58 +313,57 @@ represented by the job parameter "input.file.name":
 ----
 
 
-The main points of interest here are injecting the value of
+The main points of interest in the preceding example are injecting the value of
 `#{jobParameters['input.file.name']}`
 as the Resource property value and setting the `ItemReader` bean
-to be of __Step scope__ to take advantage of
-the late binding support which allows access to the
+to have __Step scope__, Setting the bean to have Step scope takes advantage of
```
Comment (mminella) [inline spring-batch-docs/asciidoc/spring-batch-integration.adoc]: Why is setting capitalized?

File: spring-batch-docs/asciidoc/spring-batch-integration.adoc [hunk 3]
```diff
@@ -315,58 +313,57 @@ represented by the job parameter "input.file.name":
...
-* `auto-startup` Boolean flag to indicate that the endpoint should start automatically on
+(The exact implementation depends on whether the component's input channel is a
+`SubscribableChannel` or `PollableChannel`.)
+* `auto-startup`: Boolean flag to indicate that the endpoint should start automatically on
 startup. The default is __true__.
-* `request-channel` The input `MessageChannel` of this endpoint.
-* `reply-channel` `Message Channel` to which the resulting `JobExecution` payload will be sent.
-* `reply-timeout` Allows you to specify how long this gateway will wait for the reply message
+* `request-channel`: The input `MessageChannel` of this endpoint.
+* `reply-channel`: `Message Channel` to which the resulting `JobExecution` payload is sent.
```
Comment (mminella) [inline spring-batch-docs/asciidoc/spring-batch-integration.adoc]: Remove space between Message and Channel

File: spring-batch-docs/asciidoc/spring-batch-integration.adoc (hunk 1 above)
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/spring-batch-integration.adoc]: Thank you.

File: spring-batch-docs/asciidoc/spring-batch-integration.adoc (hunk 2 above)
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/spring-batch-integration.adoc]: I meant to make it be two sentences. The comma before "Setting" should be a period. Thans for catching it.

File: spring-batch-docs/asciidoc/spring-batch-integration.adoc (hunk 3 above)
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/spring-batch-integration.adoc]: Yes, please. Thanks.

**Comment by cppwfs** [comment]
Applied code review changes mentioned above.  Rebased, Squashed, Merged.

**Pull request closed without merging**